.github/workflows/ocwm-pre-meeting-check.yml

```
@@ -0,0 +1,83 @@
+name: OCWM Pre-Meeting Check
+
+on:
+  schedule:
+    - cron: '50 18 15-21 * 1'  # Runs at 11:50 AM PT on the 3rd Monday of the month
```

```suggestion
    - cron: '50 21 * * 1'  # Runs at 21:50 UTC, which is 10 minutes before 14:00 PT every Monday
```

My apologies but the code I suggested to be added was not working as I was expecting so sorry for suggesting it. Let's return your previous change.

Make the process runs only the third monday is not possible only using cron. We'll need to add a new step the workflow. I'll suggest the change later.